content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
++++
+title = "Kubeflow on-prem in k8s multi-node cluster"
+description = "How to install Kubeflow on-prem leveraging Dynamic Volume Provisioning and NFS volumes"
+weight = 30
++++
+
+This guide describes how to setup Kubeflow on premise in a multi-node cluster using dynamic volume provisioning.
+
+## Vanilla on-prem KubeFlow installation
+
+In order to install KubeFlow in a on-prem Kubernetes cluster (both single node and multi-node) you can follow this guide
+[Kubeflow on Kubernetes](/docs/started/getting-started-k8s).
+
+At the end of the installation, as stated in the troubleshooting section [Pods stuck in Pending state](/docs/other-guides/troubleshooting/#pods-stuck-in-pending-state), you could have some Persistent Volume Claims (PVCs) ubounded.
```

Comment:
It's better to specify that this solution is feasible for single node cluster only.
I would like to write for example:
"To fix this issue, in case of single node Kubernetes cluster, see the troubleshooting section.."
What do you think?